File: MapAYakApi/Models/Dtos/Data/RouteDto.cs
namespace MapAYakApi.Dtos.Data
{
    public class RouteDto
    {
        public string UserName { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public IEnumerable<CoordinateDto> Coordinates { get; set; }
    }
}

File: MapAYak/src/app/core/dtos/map/route.dto.ts
import { CoordinateDto } from "./coordinate.dto";

export class RouteDto {
    public userName: string;
    public name: string;
    public description: string;
    public coordinates: CoordinateDto[];

    public constructor(initial?: Partial<RouteDto>) {
        Object.assign(this, initial);
    }
}
